apps/server/src/modules/news/uc/news.uc.spec.ts

```diff
@@ -40,6 +42,40 @@ describe('NewsUc', () => {
 			id: courseTargetId,
 		},
 	};
+	const federalState = new FederalStateEntity({
+		name: 'string',
+		abbreviation: 'string',
+		logoUrl: 'string',
+		createdAt: new Date(),
+		updatedAt: new Date(),
+	});
+
+	const school = new SchoolEntity({
+		_id: schoolId,
+
+		name: 'string',
+		federalState,
+	});
+	const creator: User = {
+		_id: new ObjectId(userId),
+		id: userId,
+		email: 'string',
+		firstName: 'string',
+		lastName: 'string',
+		roles: new Collection<Role>(User),
+		school,
+		createdAt: new Date(),
+		updatedAt: new Date(),
+		resolvePermissions: () => ['a'],
+	};
+	const exampleNews: TeamNews = new TeamNews({
+		title: 'string',
+		content: 'string',
+		displayAt,
+		creator,
+		school: schoolId,
+		target: teamTargetId,
+	});
```

Comment:
To be honest it's a bit weird seeing this kind of code here, but I know you've just expanded the previous solution so I won't request the refactor of it here and now. But for the future please remember to try to put these kind of things into some factory functions or some helper functions to not have too many objects like that just at the very top of the main testing suite function.

Reply:
removed